chrome_extension_starfish_ide_from_scratch/content.js

```diff
@@ -0,0 +1,57 @@
+
+current_url=window.location.href;
+var page_uri ="ide://#"+window.location.href;
+ 
+if(current_url.indexOf("https://github.com/")!=-1)
+
+{//Code for Github Begins
+
+  if(!localStorage.getItem("starfish")){
+     navigator.registerProtocolHandler("web+ide",
+                           "https://github.com/?ide=%s",
+                           "ide");
+     localStorage.setItem("starfish","true");
+  }
+var link = document.createElement('a');
+link.setAttribute("title","Open In IDE");
+//var page_uri ="ide://#"+window.location.href;
+link.setAttribute("href",page_uri);
+link.setAttribute("class","btn btn-sm btn-primary dev")
+link.innerHTML="Open in IDE";
+
+var starfish=document.createElement('div');
+//starfish.setAttribute("class","empty-icon position-relative gitpod-nav-btn float-right");
+starfish.appendChild(link);
+var el=document.getElementsByClassName("file-navigation")[0];
+el.appendChild(starfish); 
+
+} //Code for Github Ends here
+if(current_url.indexOf("https://gitlab.com")!=-1)
```

Comment:
why not just a "else if" ?

Reply:
Sure,that would make the code more efficient.

Meanwhile I am thinking to develop on bitbucket as well.
Some other websites you would like me to work on??

Reply:
gist.github.com ?

Reply:
okay...i'll work on it as well

Reply:
> why not just a "else if" ?

fixed on latest [commit](https://github.com/maxandersen/starfish/pull/1/#discussion_r407161132)